kubeflow/manifests
# feat(spark): integrate Spark operator in Kubeflow manifests

This PR introduces the integration of the Spark operator into the Kubeflow manifests, enabling users to run distributed Spark workloads directly within their Kubeflow environment. With this addition, Kubeflow users can leverage Spark for large-scale data processing as part of their machine learning (ML) pipelines, improving scalability and efficiency.

#### **Key Changes**:
- Added Spark operator manifests to support Spark job execution.
- Ensured integration of the Spark operator with Kubeflow pipelines, allowing users to submit and monitor Spark jobs within the Kubeflow UI.
- Updated the documentation to include instructions on how to deploy and use the Spark operator within Kubeflow.
- Configured role-based access control (RBAC) to secure Spark job submissions and resource usage.
  
#### **Benefits**:
- Enables distributed data processing using Spark directly within Kubeflow.
- Simplifies the management and orchestration of Spark jobs as part of ML workflows.
- Provides an easy-to-use interface to monitor and manage Spark workloads.

#### **How to Test**:
1. Deploy the updated Kubeflow manifests, including the Spark operator.
2. Submit a sample Spark job via the Kubeflow pipeline UI.
3. Verify the job execution, logs, and results within the Spark UI and Kubeflow pipeline dashboard.
